ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -304,6 +327,24 @@ const CreateRunPage = () => {
     defaultValues: defaultValues,
   });
 
+  const {
+    fields: pFields,
+    append: pAppend,
+    remove: pRemove,
+  } = useFieldArray({
+    name: 'jobConfigs.parameters',
+    control: form.control,
+  });
+
+  const {
+    fields: lFields,
+    append: lAppend,
+    remove: lRemove,
```

Comment:
I'd use `parametersFields` and `labelsFields` for these so it's easier to comprehend what they are.